locale/en/blog/vulnerability/july-2022-security-releases.md

```diff
@@ -27,8 +27,21 @@ The 14.x release line of Node.js is vulnerable to three medium severity issues a
 
 Releases will be available on, or shortly after, Tuesday, July 5th, 2022.
 
+However, when details of the OpenSSL defects are released on the 5th, our team will be making a more detailed assessment
+on the likely severity for Node.js users.
+
+The OpenSSL release may delay the Node.js release date. See [OpenSSL Security Release](#update-04-07-2022-openssl-security-release)
+
+Please monitor the **nodejs-sec** Google Group for updates, including a decision within 24 hours after the OpenSSL release regarding release timing, and full details of the defects upon eventual release: https://groups.google.com/forum/#!forum/nodejs-sec
+
 ## Contact and future updates
 
 The current Node.js security policy can be found at https://nodejs.org/en/security/. Please follow the process outlined in https://github.com/nodejs/node/blob/main/SECURITY.md if you wish to report a vulnerability in Node.js.
 
 Subscribe to the low-volume announcement-only nodejs-sec mailing list at https://groups.google.com/forum/#!forum/nodejs-sec to stay up to date on security vulnerabilities and security-related releases of Node.js and the projects maintained in the nodejs GitHub organization.
+
+## (Update 04-07-2022) OpenSSL Security Release
```

Comment:
That looks like "April 7" rather than "July 4" to Americans and probably many others.

```suggestion
## (Update 04-Jul-2022) OpenSSL Security Release
```